apps/Learning-Buddy/navigation/RootNavigator.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { useState } from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { AuthStack } from './AuthStack';
+import { AppStack } from './AppStack';
+
+export const RootNavigator = () => {
+  const { user } = useState();
```

Comment:
Should be in a proper useState format eg.  const [user, setUser] = useState(initialstate);

Reply:
fixed